tasks/rh6.yml

```diff
@@ -3,6 +3,7 @@
     name=kernel
     state=latest
   register: kernel
+  when: update_kernel == true
```

Comment:
What about:
`when: update_kernel is defined and update_kernel | bool`

It should be safer.